### DISCUSSION TITLE
Gnful universal lists?

### DISCUSSION THREAD

**Discussion by mathieureguer** [koeberlin/Latin-Character-Sets#16] [General]
Having universal glyph lists with unicodes, names and hex is really handy. Maybe having an additional set of lists that are [gnful](https://github.com/LettError/glyphNameFormatter) compatible would be useful? 

I am happy to build these, if you think it is worth adding to the repo or course.

(Also: Latin S, M, L, XL are such useful ressources. Thank you for this massive yet essential piece of work @koeberlin)

**Comment by koeberlin** [comment]
Thanks a lot for your kind words, Mathieu! 🙏 They mean a lot, and it’s good to know the character sets can be helpful.
Thanks also for your reasonable suggestion and offer. I need to fix my github connection and intergrate some changes, then I’d gladly come back to it!

**Reply by LettError** [reply]
Gnufl team expresses interest.

**Reply by justvanrossum** [reply]
This looks Gnufled: https://github.com/koeberlin/Latin-Character-Sets/tree/main/CharacterSets/RoboFont
